app/components/main-content/LandingPageTreeVisualizationPanel.tsx

```diff
@@ -1,64 +1,12 @@
 'use client'
-import LottieSeriesPlayer from "../common/LottieSeriesPlayer";
-import deleteNodeAnimationData from "../../../public/lottiefiles/delete-node.json";
-import editNodeAnimationData from "../../../public/lottiefiles/edit-node.json";
-import dragTreeAnimationData from "../../../public/lottiefiles/drag-tree.json";
-import addNodeAnimationData from "../../../public/lottiefiles/add-node.json";
-import promptInputAnimationData from "../../../public/lottiefiles/prompt-input.json";
-import treeGenerateAnimationData from "../../../public/lottiefiles/tree-generate.json";
-
-import LinktaLogoWithText from "@/app/components/layout/LinktaLogoWithText";
-
-export interface LottieAnimation {
-  src: object;
-  duration: number;
-}
-
-const editNodeAnimation: LottieAnimation = {
-  src: editNodeAnimationData,
-  duration: 3800
-};
-
-const dragTreeAnimation: LottieAnimation = {
-  src: dragTreeAnimationData,
-  duration: 1800
-};
-
-const addNodeAnimation: LottieAnimation = {
-  src: addNodeAnimationData,
-  duration: 2900
-};
-
-const promptInputAnimation: LottieAnimation = {
-  src: promptInputAnimationData,
-  duration: 2000
-};
-
-const treeGenerateAnimation: LottieAnimation = {
-  src: treeGenerateAnimationData,
-  duration: 2000
-};
-
-const deleteNodeAnimation: LottieAnimation = {
-  src: deleteNodeAnimationData,
-  duration: 800 
-};
-
+import Spline from '@splinetool/react-spline';
 
 export default function LandingPageTreeVisualizationPanel() {
-  const treeAnimationData: LottieAnimation[] = [
-    promptInputAnimation,
-    treeGenerateAnimation,
-    dragTreeAnimation,
-    editNodeAnimation,
-    deleteNodeAnimation,
-    addNodeAnimation,
-  ]
   
   return (
     <div>
-      <div style={{ maxWidth: '320px' }}>
-        <LottieSeriesPlayer animationData={treeAnimationData} />
+      <div className="absolute">
```

Comment:
Consider adding an aria label for spline: 
```
aria-label="Interactive 3D model of a tree, which you can rotate and click to explore different functionalities. "
```

Reply:
I saw the spline component comes with several event handlers, do we plan to incorporate those? If we plan to add extra code related to the spline scene, it is probably better to create a separate component for the spline scene and import it into this current component - so we can minimize merge conflicts and styling doesn't need to wait for the spline functionalities to be implemented first.

Reply:
Good call on the aria label. Fixed the size change bug caused by spline. Spline has the responsive feature, but it's buggy. Will need to configure different sizes for different screens once we have the breakpoints established.

Will probably not use any react-spline event handlers at the moment and this will be our only spline scene for now. All minor event configurations were handled in spline editor and exported as is. 
Will explore necessity to create a separate spline component in the future.